<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XML Generator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-run">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="srcLabel">
      <property name="text">
       <string>Source file</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="srcLayout">
      <item>
       <widget class="QLineEdit" name="srcEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="srcButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="document-open">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="destLabel">
      <property name="text">
       <string>Destonation path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="destLayout">
      <item>
       <widget class="QLineEdit" name="destEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="destButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="document-open">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="encLabel">
      <property name="text">
       <string>Text encoding</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="encCombo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="indexLabel">
      <property name="text">
       <string>Path index</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="indexCombo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="rootLabel">
      <property name="text">
       <string>Root class</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="rootEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="dataLabel">
      <property name="text">
       <string>Root data</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="dataEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>srcEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validateDialogParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>879</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validateDialogParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>880</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rootEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validateDialogParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>861</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indexCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>validateDialogParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>srcButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>srcButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>destButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validateDialogParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validateDialogParams()</slot>
  <slot>srcButtonClicked()</slot>
  <slot>destButtonClicked()</slot>
  <slot>generateButtonClicked()</slot>
 </slots>
</ui>
